tests/test_pfx.c

```diff
@@ -363,12 +370,37 @@ static void test_issue99(void)
 	pfx_table_free(&pfxt);
 }
 
+static void test_issue152(void)
+{
+	struct pfx_table pfxt;
+	struct pfx_record *records = calloc(6, sizeof(struct pfx_record));
```

Comment:
unsure myself, but why not use `struct pfx_record records[6];` instead?

Reply:
No specific reason, I want initialized memory there and apparently calloc occurred to me before memset did.